### DISCUSSION TITLE
List of all ports used by the LTSP server

### DISCUSSION THREAD

**Discussion by mavrinac** [ltsp/ltsp#772] [General]
What are all the ports required by all the services for a working LTSP server using either one or two nic setup? e.g. dhcp, dnsmasq, epoptes, etc. The security configuration of a campus firewall seems to be blocking some things between our LTSP subnets.

**Comment by alkisg** [comment]
Unfortunately NFS and Epoptes aren't firewall-friendly.
See https://wiki.debian.org/SecuringNFS for how to configure NFS to be firewall-friendly; its ports are also listed there.
Epoptes currently doesn't have a similar method to make it firewall friendly; opening ports 789 and 5499-5501 will only allow some of its functionality.
dnsmasq usually needs 53, 67, 68, 69 and 4011.
ssh and sshfs need port 22.

**Comment by mavrinac** [comment]
Thanks
